### PR TITLE
Add support for querying pairwise compatibility status for a given package

### PR DESCRIPTION
**Pull request by liyanhui1228**
- Add support in compatibility_store for querying whether a given package is compatible with all the rest of google owned python packages.

### CODE REVIEW

File: compatibility_lib/compatibility_lib/utils.py [hunk 1]
```diff
@@ -95,9 +95,11 @@ def _get_from_bigquery(self, package_name):
             a dict mapping from dependency package name (string) to
             the info (dict)
         """
-        if self.store is not None and package_name in configs.PKG_LIST:
-            depinfo = self.store.get_dependency_info(package_name)
-            return depinfo
+        if self.store is not None:
+            if package_name in configs.PKG_LIST or \
+                            package_name in configs.WHITELIST_URLS:
```
Comment (ylil93) [inline compatibility_lib/compatibility_lib/utils.py]: Use parentheses http://google.github.io/styleguide/pyguide.html#32-line-length

File: compatibility_lib/compatibility_lib/compatibility_store.py [hunk 2]
```diff
@@ -360,6 +358,55 @@ def get_compatibility_combinations(self,
...
+        for row in results:
+            install_name_lower, install_name_higher, _, _, _, _ = row
+            p_lower = package.Package(install_name_lower)
+            p_higher = package.Package(install_name_higher)
+            if not packages_to_results.get(frozenset([p_lower, p_higher])):
+                packages_to_results[frozenset([p_lower, p_higher])] = []
+            packages_to_results[frozenset([p_lower, p_higher])].append(
+                self._row_to_compatibility_status([p_lower, p_higher], row)
+            )
+        return {p: crs for (p, crs) in packages_to_results.items()}
```
Comment (ylil93) [inline compatibility_lib/compatibility_lib/compatibility_store.py]: Is this necessary? Why not just do:
```
return packages_to_results
```

File: compatibility_lib/compatibility_lib/compatibility_store.py [hunk 3]
```diff
@@ -360,6 +358,55 @@ def get_compatibility_combinations(self,
...
+                cursor.execute(
+                    query, (install_names_lower, install_names_higher,
+                            package_name, package_name))
+                results = cursor.fetchall()
+
+        for row in results:
+            install_name_lower, install_name_higher, _, _, _, _ = row
+            p_lower = package.Package(install_name_lower)
+            p_higher = package.Package(install_name_higher)
+            if not packages_to_results.get(frozenset([p_lower, p_higher])):
```
Comment (ylil93) [inline compatibility_lib/compatibility_lib/compatibility_store.py]: I think you can improve readability if you do
```
key = frozenset([p_lower, p_higher])
```

Comment (liyanhui1228) [reply compatibility_lib/compatibility_lib/compatibility_store.py]: Done.

File: compatibility_lib/compatibility_lib/compatibility_store.py (hunk 2 above)
Comment (liyanhui1228) [reply compatibility_lib/compatibility_lib/compatibility_store.py]: Done.

File: compatibility_lib/compatibility_lib/utils.py (hunk 1 above)
Comment (liyanhui1228) [reply compatibility_lib/compatibility_lib/utils.py]: Done.

**Pull request merged**